runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankItemCache.java

```diff
@@ -0,0 +1,23 @@
+package net.runelite.client.plugins.banktags;
+
+import net.runelite.api.Item;
+
+public class BankItemCache {
```

Comment:
Remove this class, and just inject client and call client.getItemContainer (or w/e it is named atm) in tag manager